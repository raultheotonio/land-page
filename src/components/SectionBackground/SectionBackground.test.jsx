import { screen } from '@testing-library/react';
import { SectionBackground } from '.';
import renderTheme from '../../utils/RenderTheme';

describe('<SectionBackground />', () => {
  it('should render with background dark', () => {
    const { container } = renderTheme(
      <SectionBackground background>
        <h1>children</h1>
      </SectionBackground>
    );

    expect(screen.getByRole('heading')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });

  it('should render', () => {
    renderTheme(
      <SectionBackground background={false}>
        <h1>children</h1>
      </SectionBackground>
    );

    expect(screen.getByRole('heading')).toBeInTheDocument();
  });
});
