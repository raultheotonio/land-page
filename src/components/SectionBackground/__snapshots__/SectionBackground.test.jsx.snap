// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionBackground /> should render with background dark 1`] = `
<div>
  <div
    class="sc-bcXHqe lawjBw"
  >
    <div
      class="sc-gswNZR DuAtq"
    >
      <h1>
        children
      </h1>
    </div>
  </div>
</div>
`;
